### ISSUE TITLE
Training with sdf loss

### ISSUE THREAD

**Issue by mkrishnan9** [maturk/dn-splatter#67]
Hello, I was trying to run your code with the sugar style sdf loss turned on, with the replica dataset. However I noticed that the runtime is really high and the number of gaussians keeps increasing. Is this what I should expect? Can you please share the command you used to run the sugar comparisons in your paper?

**Comment by maturk** [comment]
Hey @mkrishnan9 , the SuGaR style SDF loss is an experimental feature. The training time is long since it requires a pretty costly KNN search of all Gaussians in the scene at fixed intervals. 

Since the code related to the SuGaR style SDF loss is experimental, I have decided to stop supporting the feature in the latest updates in #64 . 

Regarding the results in the paper, these concern the SuGaR meshing strategy proposed in the Sec 4.2 from the official SuGaR paper. The meshing strategy does not depend on the SuGaR SDF style loss. Instead, the SuGaR meshing strategy is implemented in the `export_mesh.py` script, specifically starting from this [line](https://github.com/maturk/dn-splatter/blob/811f5cdf342d92d12a9d9a4be185d374c430c184/dn_splatter/export_mesh.py#L469).

To extract a mesh using the SuGaR meshing strategy, use the command `gs-mesh sugar-coarse ...`. This will do all the logic of shooting rays into the scene, querying the local Gaussian density, and then extracting level sets based on some thresholds.

Let me know if you have any more questions/doubts. We are in the process of updating this github repo with some new features, but I have also decided to stop supporting some very experimental stuff.
